components/core/core-repository/src/main/java/org/eclipse/dirigible/components/repository/RepositoryConfig.java

```diff
@@ -10,20 +10,26 @@
  */
 package org.eclipse.dirigible.components.repository;
 
+import java.io.File;
+import org.eclipse.dirigible.commons.config.Configuration;
 import org.eclipse.dirigible.commons.config.StaticObjects;
 import org.eclipse.dirigible.repository.api.IRepository;
 import org.eclipse.dirigible.repository.local.LocalRepository;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 
-@Configuration
+@org.springframework.context.annotation.Configuration
 public class RepositoryConfig {
 
+
     @Bean
     public IRepository repository() {
-        LocalRepository localRepository = new LocalRepository("target");
+        String repoFolderPath = Configuration.get(Configuration.DIRIGIBLE_REPOSITORY_LOCAL_ROOT_FOLDER, "target");
+        boolean absolutePath = repoFolderPath.startsWith(File.separator);
```

Comment:
let use the explicit DIRIGIBLE_REPOSITORY_LOCAL_ROOT_FOLDER_IS_ABSOLUTE here as to work on any filesystem

Reply:
It uses the files separator to check whether it is absolute.  This way it should be working on every os or I'm missing something?

Reply:
on Windows you could have something like `C:\MyFolder` which will not be taken as an absolute

Reply:
Somehow I forgot about the Windows :)
I will fix it on Monday.